DOC-1102: Add information about the `system` database (#14304)
cockroachdb/docs: _includes/v21.1/sql/preloaded-databases.md

> New clusters and existing clusters [upgraded](upgrade-cockroach-version.html) to v2.1 or later will include auto-generated databases, with the following purposes:

Author: I think if it's true to say CRDB v2.1, then it's also true to say whatever the page version it is on, so I'll use `{{ page.version.version }}` here instead.

- The empty `defaultdb` database is used if a client does not specify a database in the [connection parameters](connection-parameters.html).
- The `movr` database contains data about users, vehicles, and rides for the vehicle-sharing app [MovR](movr.html).

> - An empty database called `postgres` is provided for compatibility with PostgreSQL client applications that require it.

Reviewer: ```suggestion
- The empty `postgres`database is provided for compatibility with PostgreSQL client applications that require it.
```
